docs/sources/auth/azuread.md

```diff
@@ -0,0 +1,103 @@
++++
+title = "Azure AD OAuth2 Authentication"
+description = "Grafana OAuthentication Guide "
+keywords = ["grafana", "configuration", "documentation", "oauth"]
+type = "docs"
+[menu.docs]
+name = "Azure AD"
+identifier = "azuread_oauth2"
+parent = "authentication"
+weight = 3
++++
+
+# Azure AD OAuth2 Authentication
+
+To enable the Azure AD OAuth2 you must register your application with Azure AD. Azure AD will generate a client ID and secret key for you to use.
+
+## Create Azure AD application
+
+1. Log in to portal.azure.com and click "Azure Active Directory" in the side menu, then click the "Properties" sub-menu item.
+
+2. Click "App Registrations" and add a new application registration:
+   - Name: Grafana
+   - Application type: Web app / API
+   - Sign-on URL: `https://<grafana domain>/login/azuread`
+
+3. Click the name of the new application to open the application details page.
+
+4. Click "Endpoints"
```

Comment:
```suggestion
4. Click **Endpoints**.
```

Reply:
I changed all the clickable items to be bold.